CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2024-07-16
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/). This release is titled as v20240716.
+
+### Announcements
+
+* 1.30 is the next LTS version after 1.27. Upgrade from 1.27 LTS to 1.30 LTS will be possible starting August 2024. More information about AKS LTS is available [here](https://learn.microsoft.com/azure/aks/long-term-support).
+* 1.27 community supported has ended. To exit [Long-Term Support (LTS)](https://learn.microsoft.com/azure/aks/long-term-support) upgrade to 1.28 and disable LTS with the `az aks upgrade` command.
```

Comment:
nit/question: do you need to disable LTS before upgrade? Will you be charged if you upgrade and don't disable? Is there validation to prevent and ordered process? 

Not blocking as docs should have this clarified more than release notes

Reply:
@chasewilson - will take note of this and try and add it to docs. I don't know if you will be charged for this.